bindsnet/conversion/conversion.py

```diff
@@ -197,7 +197,8 @@ def set_batch_size(self, batch_size) -> None:
 class PassThroughNodes(nodes.Nodes):
     # language=rst
     """
-    Layer of `integrate-and-fire (IF) neurons <http://neuronaldynamics.epfl.ch/online/Ch1.S3.html>`_ with using reset by
+    Layer of `integrate-and-fire (IF) neurons
+    <http://neuronaldynamics.epfl.ch/online/Ch1.S3.html>`_ with using reset by
     subtraction.
```

Comment:
is this accurate?

Reply:
No, it is not. Why not move these nodes to nodes.py?

Reply:
>Also, why not move these nodes to nodes.py?

I imagine it depends on the intent of the author. If it is a general node that lots of 3rd party consumers might want to use, it makes sense to put it in the `nodes` module. If it is a one off node that is specific to the task at hand, and does not have much utility to consumers, it may be better to keep it local to this file and _not_ share it as part of the public API, to keep the public API tight and simple.

Reply:
I think the subtractive IF neurons is a general node

Reply:
I think the subtractive IF neurons are specific to the `conversion` module, actually. But yeah, the doc string is wrong.